documentation/how-to/digitize.en.md

```diff
@@ -128,6 +128,14 @@ The reshape tool allows you to change line and polygon geometries by drawing sha
 result in a reshaped line following the edge of the drawn shape or a polygon with the drawn shaped
 used to clip or expand the geometry.
 
+## Reshape eraser tool
+
+The reshape eraser tool is a feature in the geometry editor, designed to facilitate the removal of parts of a line or polygon geometry. This tool enables to efficiently correct paths or polygons directly within the application interface and directly from the field conditions.
+
+Suppose you have a polygon representing a building footprint, but there's an unwanted protrusion. You can use the erase tool to precisely remove that portion of the polygon, ensuring accuracy in your geometry representation.
```

Comment:
I'd remove that line entirely, just stick to a on paragraph description + the webm.

Reply:
Sorry for the delayed response. Thank you, @nirvn